### ISSUE TITLE
Clicking "More options" in the chat reaction menu causes menu to disappear

### ISSUE THREAD

**Issue by cbeams** [bisq-network/bisq2#2507]
## Environment

```
$ git show
commit 96b04663ddcf55b82dca1fb0a1caa106aaac4304 (HEAD -> main, origin/main, origin/HEAD)
Merge: 06dc51000 5c2b2eff8
Author: djing-chan <146422224+djing-chan@users.noreply.github.com>
Date:   Sun Jul 28 18:09:01 2024 +0700

    Merge pull request #2503 from HenrikJannsen/cleanup-deprecated-code

    Cleanup deprecated code
```

## Repro steps

1. Navigate to any existing chat message
2. Click the reactions icon to bring up the reactions menu
3. Click the "More options" ellipsis as shown in the screenshot
4. Notice that the reactions menu disappears completely

<img width="446" alt="Screenshot 2024-07-28 at 3 14 16 PM" src="https://github.com/user-attachments/assets/584a71a0-d559-4774-bf66-e28bafe9c467">

## Expected behavior

In (4) above, an expanded menu of all available reaction emojis should appear.

**Comment by cbeams** [comment]
> Do you mean that you expect the ellipsis menu to give you more options to the reactions?

Yes, thanks, that is what I expect, as illustrated in the Signal screenshots below.

1. Before clicking the  "More" ellipsis in Signal
<img width="846" alt="image" src="https://github.com/user-attachments/assets/5ce4df6b-a35a-4b7d-a3da-9f4266d17836">

2. After clicking the ellipsis in Signal
<img width="352" alt="image" src="https://github.com/user-attachments/assets/2751e9a7-6fc3-4df9-b8e8-957440afa15e">

> The ellipsis menu gives more action options to the message (it's not related to reactions)

Ok. I think this will for sure cause user confusion, because it's such a strong convention in other chat apps that if one sees a "More" affordance in the context of reactions it means "bring up the full menu of reactions" as seen above.

And the likelihood of such confusion is further increased by the fact that the ellipsis menu does not actually "give more action options to the message"; it just does nothing and the reaction menu disappears entirely. The behavior right now simply feels like a bug.

If the More options ellipsis did actually "give more action options to the message", then users would at least understand what's going on, granted, but I think it would be a disappointing experience, as they would have gotten their hopes up about being able to choose the perfect emoji and then had those hopes dashed when they realize the ellipsis didn't mean what they (understandably) assumed it meant.

> At the moment, there's only those six reactions.

Ok, for however long this is the case, I'd suggest just removing the ellipsis button entirely when the reactions menu is visible. GitHub offers a similarily limited menu of possible reactions, and the absence of any "more" affordance makes that limitation quite obvious to the user. The user might wish there were more options, but at least they don't go through the negative experience of thinking that there are going to be more and then getting disappointed when there aren't.

<img width="443" alt="image" src="https://github.com/user-attachments/assets/eba82260-3573-4ca7-b700-926164734fd1">

**Comment by iMac7** [comment]
@axpoems this still available?

**Comment by HenrikJannsen** [comment]
@iMac7 Sorry for the late reply. Yes if the issues is still reproducable on main branch, its still open. Do you want to work on it?

**Comment by axpoems** [comment]
This is by design. To fully implement cbeams suggestion we would need another design approach.
I close the issue for now. If needed we can open another one to continue adding more reactions.
